pandas/tests/dtypes/test_dtypes.py

```diff
@@ -475,7 +476,8 @@ def test_basic(self, dtype):
 
     def test_empty(self):
         dt = PeriodDtype()
-        with pytest.raises(AttributeError):
+        msg = "object has no attribute 'freqstr'"
```

Comment:
I _think_ this is a bug, see #27388. so maybe add a comment.
```suggestion
        # https://github.com/pandas-dev/pandas/issues/27388
        msg = "object has no attribute 'freqstr'"
```